### ISSUE TITLE
Feed me plugin - clicking [save] or [save and continue] button on field mapping screen does nothing.

### ISSUE THREAD

**Issue by mach1media** [craftcms/feed-me#536]
### Description

On Feed Me field mapping screen clicking [save] or [save and continue] does nothing.

I'm also noticing on this field mapping screen that nothing happens after selecting a field from any of the dropdowns on the page.

I'm using an RSS feed from another Craft site running in my local environment. The feed outputs properly in Chrome.

Mac OS X 10.14
Safari 12.1
Chrome 74.0.3729.131 

### Steps to reproduce

1. Create a new feed
2. Advance to the field mapping screen
3. Select an RSS node from any of the field mapping dropdowns
4. Click [save]

### Additional info

- Craft version: 3.1.26 (Solo)
- PHP version: 7.3.1
- OS version: Darwin 18.5.0
- Database driver & version: MySQL 5.7.25
- Image driver & version: GD 7.3.1
- Plugins & versions: 
    - Entry Instructions 1.0.4
    - Feed Me 4.1.0
    - Field Manager 2.1.0
    - Image Resizer 2.0.4
    - Position Fieldtype 1.0.14
    - Redactor 2.3.3.2
    - Redactor Tweaks 2.0.2
    - Smith 1.0.0
    - Super Table 2.1.19

**Comment by mach1media** [comment]
Console errors on /admin/feed-me/feeds/map/1

jquery.js:3841 jQuery.Deferred exception: Cannot read property 'item_' of undefined TypeError: Cannot read property 'item_' of undefined
    at HTMLSelectElement.<anonymous> (http://kkresg.local/cpresources/9bb12296/js/feed-me.min.js?v=1557791866:1:43865)
    at HTMLSelectElement.dispatch (http://kkresg.local/cpresources/7513f95/jquery.js?v=1557347328:1:42344)
    at HTMLSelectElement.v.handle (http://kkresg.local/cpresources/7513f95/jquery.js?v=1557347328:1:40356)
    at Object.trigger (http://kkresg.local/cpresources/7513f95/jquery.js?v=1557347328:1:70447)
    at HTMLSelectElement.<anonymous> (http://kkresg.local/cpresources/7513f95/jquery.js?v=1557347328:1:71042)
    at Function.each (http://kkresg.local/cpresources/7513f95/jquery.js?v=1557347328:1:2769)
    at k.fn.init.each (http://kkresg.local/cpresources/7513f95/jquery.js?v=1557347328:1:1418)
    at k.fn.init.trigger (http://kkresg.local/cpresources/7513f95/jquery.js?v=1557347328:1:71018)
    at HTMLDocument.<anonymous> (http://kkresg.local/cpresources/9bb12296/js/feed-me.min.js?v=1557791866:1:44050)
    at e (http://kkresg.local/cpresources/7513f95/jquery.js?v=1557347328:1:29297) undefined
k.Deferred.exceptionHook @ jquery.js:3841

jquery.js:3850 Uncaught TypeError: Cannot read property 'item_' of undefined
    at HTMLSelectElement.<anonymous> (feed-me.js:3795)
    at HTMLSelectElement.dispatch (jquery.js:5233)
    at HTMLSelectElement.v.handle (jquery.js:5040)
    at Object.trigger (jquery.js:8461)
    at HTMLSelectElement.<anonymous> (jquery.js:8539)
    at Function.each (jquery.js:367)
    at k.fn.init.each (jquery.js:202)
    at k.fn.init.trigger (jquery.js:8538)
    at HTMLDocument.<anonymous> (feed-me.js:3811)
    at e (jquery.js:3557)

**Comment by tommers-walker** [comment]
Same here :(

**Comment by Christopholus** [comment]
Getting the same issue here as well. All of the "Feed element" mapping fields appear to be blank (and the select boxes seem to have reverted to browser default selectboxes)

**Comment by Christopholus** [comment]
I've fixed the issue on my end. The issue was caused by having an Entry Picker field attached to the asset volume - specifically an entry picker field with no "Source" selected. Once I selected a source for the entry picker field, Feedme's "Feed element" mapping fields came back to life. In my case this issue was caused purely by user error.
That said, if there was any improvement to be made on Feedme's end, I would suggest looking into validating entry picker fields having a valid entry source to prevent similar issues from appearing in the future. 
Hope this helps somebody!
